# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../at76c503a/linux-2.6.18.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur at76c503a-orig/at76c503.c at76c503a/at76c503.c
--- at76c503a-orig/at76c503.c	2000-01-01 01:00:00.000000000 +0200
+++ at76c503a/at76c503.c	2006-09-20 22:07:08.000000000 +0300
@@ -94,7 +94,6 @@
 #include <linux/spinlock.h>
 #include <linux/list.h>
 #include <linux/smp_lock.h>
-#include <linux/devfs_fs_kernel.h>
 #include <linux/usb.h>
 #include <linux/netdevice.h>
 #include <linux/if_arp.h>
